### PR TITLE
done review section

### PR DESCRIPTION
**Pull request by screpyy96**

### CODE REVIEW

File: src/_component/reviews/reviews.pug [hunk 1]
```diff
@@ -0,0 +1,19 @@
+- var sectionTitle = data.reviews.sectionTitle
+include ../section-title/section-title.pug 
+
+.column(span="row" class="review")
+  each item in data.reviews.cards
+    .review-box
```
Comment (RalucaRbc) [inline src/_component/reviews/reviews.pug:6]: I don't think this `.review-box` is doing anything here so you can delete it 🤔

File: src/_component/reviews/reviews.pug [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        img.image(src=`${item.avatarUrl}`, alt="avatar")
+      .review-name=item.name 
+      .review-date=item.date
+    .review-description
+      p=item.description
+
+.column(span="row")        
+    button.disc-green     
+        i.icon__arrow-down 
+        span.button-text
```
Comment (RalucaRbc) [inline src/_component/reviews/reviews.pug:18]: Same here, what is this `span.button-text` doing?

File: src/_component/reviews/reviews.scss [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+.review-date {
+  grid-row: 2/3;
+  grid-column: 2/3;
+  display: flex;
+  align-items: center;
+  font-weight: 500;
+  font-size: 11px;
+  color: #808080;
+  color: #1875f0;
```
Comment (RalucaRbc) [inline src/_component/reviews/reviews.scss]: You have given it 2 values for color. Which one is the one you need?

**Review by RalucaRbc — APPROVED** [review]

**Review by MariaTeo — APPROVED** [review]
👌🏼

**Pull request merged**